docs/reference/operators/add-custom-op.md

```diff
@@ -36,6 +36,20 @@ A new op can be registered with ONNX Runtime using the Custom Operator API in [o
 
 * [Using Custom Ops with TF2ONNX](https://github.com/microsoft/onnxruntime-extensions/blob/main/tutorials/tf2onnx_custom_ops_tutorial.ipynb): This notebook covers converting a TF model using an existing custom op, defining new custom ops in Python to use in conversion, and defining new custom ops in C++.
 
+## Calling a native operator from custom operator
+Since 1.12.0, onnxruntime allows its native operators to be called directly by a custom operator through [API](https://github.com/microsoft/onnxruntime/blob/ced7c2deac958391414d2bbf951f86e2fc904b05/include/onnxruntime/core/session/onnxruntime_cxx_api.h#L1156).
+
+Steps are:
+
+1. Create a native operator instance by providing type constraints and attributes.
+2. Call the operator by feeding inputs and outputs.
+3. Recycle the operator.
+
+Please see examples [here](https://github.com/microsoft/onnxruntime/blob/ced7c2deac958391414d2bbf951f86e2fc904b05/onnxruntime/test/shared_lib/custom_op_utils.cc#L210).
+
+Note that the native op will be created on the same executioin provider as the custom op.
+With the feature, custom ops are empowered by native operators not only for functionalities but also for customizations like state management, or pre-processing of inputs.
```

Comment:
Can you clarify this line? Also, it'll be useful to highlight the benefits of this feature (including this) at the top of the section.

Reply:
Updated.